auth/auth-router.js

```diff
@@ -1,11 +1,69 @@
-const router = require('express').Router();
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
 
-router.post('/register', (req, res) => {
-  // implement registration
+// Imports
+const router = require("express").Router();
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+// Model
+const Users = require("./users-model");
+
+function makeToken(user) {
+  const payload = {
+    sub: user.id,
+    username: user.username
+  };
+  const options = {
+    expiresIn: "1d"
+  };
+  const token = jwt.sign(
+    payload,
+    process.env.JWT_SECRET || "thesecret",
+    options
+  );
+  return token;
+}
+// CRUD
+router.post("/register", (req, res) => {
+  let user = req.body;
```

Comment:
You can also extract just the properties you need from the body
```suggestion
  let user = { username, password } = req.body;
```